pandas/core/generic.py

```diff
@@ -2185,6 +2185,9 @@ def _repr_data_resource_(self):
 
         .. versionadded:: 0.20.0.
 
+    table : string, default None
```

Comment:
Would accepting a TableStyle be an option instead (speaking only in openpyxl terms, not sure if xlsxwriter offers that)? I feel like that could do the same thing but also give users more power over output formatting

Reply:
Both accept a style, but of course the syntax is a bit different: openpyxl uses `"TableStyleMedium9"` as default, and xlsxwriter uses `"Table Style Medium 9"`.

Reply:
Hmm wasn't thinking about that as much as having the user pass in an actual object itself

Reply:
Do you mean something like?

    style = TableStyleInfo(name="TableStyleMedium9", showFirstColumn=False,
                           showLastColumn=False, showRowStripes=True, showColumnStripes=True)

Reply:
Right that's what I had in mind (not tied to it just asking)